frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/BasicFieldPanel.tsx

```diff
@@ -31,6 +38,14 @@ export const BasicFieldPanel = ({ searchValue }: { searchValue: string }) => {
             <FieldSection label="Basic">
               {filteredCreateBasicFields.map(({ fieldType, originalIndex }) => {
                 const shouldDisableField = isLoading
+
+                // TODO: remove when signature field is out of beta
```

Comment:
Can use a marker so that it is easier to look up todos associated with a particular feature!

```suggestion
                // TODO: FRM-2054 remove when signature field is out of beta
```